src/lib/common/tag.cpp

```diff
@@ -208,48 +208,20 @@ std::string jsonInvalidCharsTransformation(const std::string& input)
 
 /* ****************************************************************************
 *
-* startTag1 -
+* startTag -
 */
-std::string startTag1
+std::string startTag
 (
   const std::string&  indent,
   const std::string&  key,
-  bool                showKey,
-  bool                isToplevel
+  bool                isVector
 )
 {
-  if (isToplevel)
-  {
-    if (showKey == false)
-    {
-      return indent + "{\n" + indent + "  {\n";
-    }
- 
-    return indent + "{\n" + indent + "  " + "\"" + key + "\" : {\n";
-  }
+  // Empty key is legal JSON. However, doesn't use that kind of JSON in Orion,
+  // so we can this instead of a showkey boolean parameter, keeping the function
+  // signature simpler
```

Comment:
This comment is strange. Words are missing ...
My guess:

    //
    // Empty key is legal JSON. However, Orion doesn't use that 'feature',
    // so instead of a showkey boolean parameter we check the JSON key to be empty
    // to decide whether the key should be printed or not.
    // This makes the function signature a bit lighter
    //